src/app/youtube/services/search.service.ts

```diff
@@ -1,40 +1,42 @@
 import { Injectable } from '@angular/core'
-import { HttpClient } from '@angular/common/http'
 import { Subject } from 'rxjs'
-import { IVideosResponse } from '../models/videos-response.model'
+import { map, pluck } from 'rxjs/operators'
 import { IVideoItem } from '../models/video-item.model'
+import { SearchHttpService } from './search-http.service'
 
 @Injectable({
   providedIn: 'root',
 })
 export class SearchService {
-  private url = './assets/response.json'
-
-  response?: IVideosResponse
-
   videos: IVideoItem[] = []
```

Comment:
still don't see why you need it 
![image](https://user-images.githubusercontent.com/51916577/130363978-76b2ff19-b2a7-4b46-a1c9-76ef8cecc9f8.png)
it used only once

Reply:
removed 'videos' field